pipeline-scripts/buildlib.groovy

```diff
@@ -354,6 +356,53 @@ def build_puddle(conf_url, keys, Object...args) {
     return puddle_dir
 }
 
+def build_ami(major, minor, version, release, mail_list) {
+    if(major < 3 || (major == 3 && minor < 9))
+        return
+    final full_version = "${version}-${release}"
+    final param = { type, name, value ->
+        [$class: type + 'ParameterValue', name: name, value: value]
+    }
+    waitUntil {
+        try {
+            build(job: 'build%2Faws-ami', parameters: [
+                param('String', 'OPENSHIFT_VERSION', version),
+                param('String', 'OPENSHIFT_RELEASE', release),
+                param('Boolean', 'USE_CRIO', true),
+                param(
+                    'String', 'CRIO_SYSTEM_CONTAINER_IMAGE_OVERRIDE',
+                    'registry.reg-aws.openshift.com:443/openshift3/cri-o:v'
+                        + full_version)])
```

Comment:
@jupierce, @mwoodson: I believe these are the only parameters that need to be
passed at the moment, and they should not be affected (complemented, if
anything) by future changes to keep the base ami updated.

Let me know if that's not the case.